movieRecomander/backend/src/main/java/com/movierecommender/backend/comments/Comment.java

```diff
@@ -65,10 +79,11 @@ public void setText(String text) {
     }
 
     public LocalDate getCommentDate() {
```

Comment:
get-erul ar trebui sa fie tot de tipul string pentru a nu strica ceva in Hibernate / other stuff, poti sa ai un getInterpretedDate care sa intoarca un LocalDate